# Some diagnostics provided by the Ukart controller box

int32  version
int32  chipID
int32  error
int8   ctlExitAck
int8   ctlEnterAck
int8   imuCalibAck
int8   powerOFF
